src/app/pcasts/controllers/merge_account_controller.py

```diff
@@ -0,0 +1,29 @@
+from app.pcasts.utils import facebook_utils, google_utils
+from . import *
+
+class MergeAccountController(AppDevController):
+
+  def get_path(self):
+    return '/users/merge/'
+
+  def get_methods(self):
+    return ['POST']
+
+  @authorize
+  def content(self, **kwargs):
+    user = kwargs.get('user')
+    token = request.args['access_token']
+    new_platform = request.args['platform']
+
+    if new_platform == "facebook":
+      facebook_info = facebook_utils.get_me(token)
+      user = users_dao.add_facebook_login(user, facebook_info)
+    elif new_platform == "google":
+      google_info = google_utils.get_me(token)
+      user = users_dao.add_google_login(user, google_info)
+    else:
+      raise Exception("Platform "+ new_platform +" not supported yet")
```

Comment:
Use the formatting syntax for consistency. Something like
`'Platform {} not supported yet'.format(new_platform)`